pattoo/api/portal/react/src/Navigation.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import { Link } from "react-router-dom";
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.